src/main/java/com/meetme/plugins/jira/gerrit/data/IssueReviewsImpl.java

```diff
@@ -276,73 +214,19 @@ public boolean equals(Object o) {
                 return false;
             }
 
-            QueryHandlerConfigGeneric that = (QueryHandlerConfigGeneric) o;
-
-            if(!this.currentConnectionType.equals(that.currentConnectionType)) {
-                return false;
-            }
-
-            if (this.currentConnectionType.equals(GerritEventKeys.CONNECTION_TYPE_SSH)) {
-                if (currentSshPort != that.currentSshPort) return false;
-                if (currentConnectionTimeout != that.currentConnectionTimeout) return false;
-                if (!currentSshPrivateKey.equals(that.currentSshPrivateKey)) return false;
-                if (!currentSshUsername.equals(that.currentSshUsername)) return false;
-                return currentSshHostname.equals(that.currentSshHostname);
-            }
-            else {
-                if (!currentUrl.equals(that.currentUrl)) return false;
-                if (!currentUsername.equals(that.currentPassword)) return false;
-                return currentPassword.equals(that.currentPassword);
-            }
+            QueryHandlerConfig that = (QueryHandlerConfig) o;
 
+            if (currentSshPort != that.currentSshPort) return false;
```

Comment:
What about brackets for condition? 
Unfortunately, classic will cover us, if we will not set into brackets :(

Reply:
Ok